.gitignore

```diff
@@ -4,7 +4,8 @@
 
 #Dependencies
 
-
+dump.rdb
```

Comment:
Yeah, it is for Redis so just ignoring it until i figure out exactly how it's used